Question 2.js

```diff
@@ -0,0 +1,19 @@
+function fibonacciEvenSum(total){
+    let sum = 0;
+    let num1 = 1;
+    let num2 = 2;
+    let temp = 0;
+
+    while(num1 + num2 <= total){
+        temp = num1 + num2;
+        num1 = num2;
+        num2 = temp;
+
+        if(num2%2==0){
+            sum += num2;
+        }
+    }
+    console.log(sum);

```

Comment:
Why are you logging the sum here instead of where you called it like you did in Problem 1? Functions should generally be "side-effect free", meaning they can be invoked and return a value and then the caller determines what to do with it. This makes the function easier to test and far easier to predict.